keyboards/handwired/uc2/info.json

```diff
@@ -0,0 +1,120 @@
+{
+    "manufacturer": "Andy Chiu",
+    "keyboard_name": "UniversalCentury2",
+    "maintainer": "Andy Chiu",
+    "bootloader": "caterina",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": false,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["F4","F5","F6","F7","B1","B3","B2","B6"],
+        "rows": ["D1","D0","D4","C6","D7","E6"]
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0x4143"
+    },
+    "layouts": {
+        "LAYOUT_all": {
+            "layout": [
+                {"matrix":[0,0],"label":"ESC", "x": -0.5000, "y": -0.5, "w": 1, "h": 1},
```

Comment:
The key positions on this layout are incorrect. You can verify it by pressing Ctrl+Shift+I in the Configurator, and turning on fast input and spamming a key to check the order.
![image](https://user-images.githubusercontent.com/4781841/188049855-e9434c5f-5906-44da-8c3a-84d7bb5055be.png)

Reply:
Still incorrect. There should be at least one key at x=0, and at least one key at y=0, and the ordering should be from left to right.
![image](https://user-images.githubusercontent.com/4781841/188073432-7a6cdef6-65b0-407d-9cf2-ebeca2bdfd06.png)

Reply:
modified, please check again

Reply:
@AndyChiu still incorrect.

Reply:
Hi, fauxpark, Is it a sorting error? 
if is this way is Ok??
Let me know how to solve the problem, thank you!

![image](https://user-images.githubusercontent.com/1038943/192366893-30ebbe3d-4514-416d-8c36-05693b104225.png)